### DISCUSSION TITLE
Is there a way to update the template to get new functionality?

### DISCUSSION THREAD

**Discussion by kirso** [chrismwilliams/astro-theme-cactus#97] [General]
Hey just seen the added search and new post type with cover image.

Was wondering if there is a way to update the template without causing breaking changes given I customized it a little bit.

**Comment by chrismwilliams** [comment]
Hi @kirso, yes there is but as mentioned before, [here](https://github.com/chrismwilliams/astro-theme-cactus/issues/43#issuecomment-1412306012) and [here](https://github.com/chrismwilliams/astro-theme-cactus/discussions/46#discussioncomment-4881668), it depends on how you set up your local repo.

You may also find [this](https://docs.github.com/en/get-started/using-git/getting-changes-from-a-remote-repository#options-for-getting-changes) helpful as well.

Hope that helps

**Reply by kirso** [reply]
Thank you so much!
